### ISSUE TITLE
document nteract on repo2docker

### ISSUE THREAD

**Issue by jzf2101** [jupyter/repo2docker#414]
@rgbkrk I can't find our documentation on how to launch nteract on repo2docker images.  How do we do that?

**Comment by choldgraf** [comment]
yes! we should add a section for nteract here: https://repo2docker.readthedocs.io/en/latest/howto/user_interface.html
